### PR TITLE
Fixed network-spec build

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: docs/network-spec/miniprotocols.tex [hunk 1]
```diff
@@ -319,7 +319,6 @@ \subsubsection{Description}
 \subsubsection{State machine}
 \haddockref{Network.TypedProtocol.ReqResp.Type}{typed-protocols-examples/Network-TypedProtocol-ReqResp-Type.html#t:ReqResp}
 
-
```
[the remark this answers is not in the document]
Comment (coot) [reply docs/network-spec/miniprotocols.tex:322]: It should be `\#` instead of `#` in the link just above.

[the remark this answers is not in the document]
Comment (coot) [reply docs/network-spec/miniprotocols.tex:322]: I noticed that this is link is a duplicate (there's one a few lines above), it should be removed.

**Comment by bolt12** [comment]
LGTM!

**Review by bolt12 — APPROVED** [review]

**Comment by coot** [comment]
bors merge

**Pull request merged**